common/config-manager.cpp

```diff
@@ -39,6 +39,7 @@ DECLARE_SINGLETON(ConfigManager);
 
 char const *const ConfigManager::kApplicationDomain = "scummvm";
 char const *const ConfigManager::kTransientDomain = "__TRANSIENT";
+char const *const ConfigManager::kSessionDomain = "session"; 
```

Comment:
Maybe call it `"__SESSION" to avoid a clash with a potential game id that might be called "session"?